CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Fixed an issue where SDK would crash when trying to read files which are not encoded with utf-8 directly from memory. [#4081](https://github.com/demisto/demisto-sdk/pull/4081)
 * Added description field to *Assets Modeling Rules* content item. [#4005](https://github.com/demisto/demisto-sdk/pull/4005)
 * Added the `validate-conf-json` pre-commit hook, checking for structure and linked content. [#4051](https://github.com/demisto/demisto-sdk/pull/4051)
+* Fixed an issue where **lint** crashed when collecting invalid logs.
```

Comment:
@samuelFain We do not add changelogs like this anymore, use the `sdk changelog --init` to create a change log